pkg/mcp/snapshot/snapshot_test.go

```diff
@@ -39,6 +39,14 @@ type fakeSnapshot struct {
 
 func (fs *fakeSnapshot) Resources(collection string) []*mcp.Resource { return fs.resources[collection] }
 func (fs *fakeSnapshot) Version(collection string) string            { return fs.versions[collection] }
+func (fs *fakeSnapshot) Collections() []string {
+	result := make([]string, 0, len(fs.resources))
+	for collection := range fs.resources {
```

Comment:
Can you just replace this loop with result = append(result, fs.resources...)?

Reply:
It won't work. fs.resources is a map.